src/lib/datepicker/month-view.ts

```diff
@@ -88,7 +88,7 @@ export class MatMonthView<D> implements AfterContentInit, OnDestroy {
   get selected(): D | null { return this._selectionModel.getFirstSelectedDate(); }
   set selected(value: D | null) {
     if (this._selectionModel instanceof MatSingleDateSelectionModel) {
-      this._selectionModel.setSelection(value);
+      this._selectionModel.add(value);
```

Comment:
Why `add` instead of `setSelection`? I think they both do the same in the end, `setSelection` makes more sense to me though

Reply:
Not entirely. `setSelection` will not notify anyone that the value has changed. It changes it silently. I need other components to update when I set this.